### ISSUE TITLE
FCP 10.8 Stalls on M2 Ultra Studio With Blackmagic Decklink Mini 4K Monitor

### ISSUE THREAD

**Issue by FiletMyYawn** [CommandPost/FCPCafe#362]
Apple Feedback Assistant ID: FB14148177
---

**DESCRIBE THE BUG:**
Updated FCP to 10.8 on M2 Ultra Studio with Blackmagic Decklink Mini 4K Monitor (installed via PCIe on external OWC Thunderbay Flex 8). Final Cut Pro now stalls on launch ONLY when Blackmagic Decklink software (Desktop Video) is installed OR when OWC Thunderbay Flex 8 is mounted via thunderbolt.

Prior to Launching FCP 10.8 for first time - (Manual Updates)
Final Cut Pro was updated to 10.8. Pro Video Formats 2.3.1 was updated after. Driver for Blackmagic Decklink was updated to latest version, Desktop Video 14.0.1, which required accepting "Allow apps downloaded from" in System Settings - Privacy Security. 

Restarted computer and launched FCP 10.8 for first time and program stalled before having to force quit. Deleted FCP preferences, uninstalled & reinstalled FCP, same issue. Disconnected OWC Thunderbay Flex 8 from Mac, FCP launches normally. Reconnected OWC Thunderbay Flex 8, FCP stalled again. Uninstalled Blackmagic Desktop Video, reconnected OWC Thunderbay Flex 8, FCP launches normally. Though I had already done this months ago, I made sure legacy system extensions were active [](https://support.apple.com/en-us/HT213969). Upon checking privacy & security settings, I noticed FCP was not listed under camera permissions, not sure if it should be. When uninstalling Blackmagic Desktop Video, I tried to find and delete .kext files and preferences, but I might have missed some.

Attempting Same Setup on M1 Mac Book Pro - 
First, installed Pro Video Formats 2.3.1. Then updated FCP to 10.8, then updated Blackmagic Desktop Video to 14.0.1, accepted "Allow apps downloaded from" in System Settings - Privacy Security, and restarted. Plugged OWC Thunderbay Flex 8 into M1 Mac Book Pro and FCP launches normally. Successfully updated a project library to new FCP version. Video output from Blackmagic Decklink to external color grading monitor is successful. Noticed FCP was listed under camera permissions, but maybe that's because the Mac Book has a built in camera?

---

**TO REPRODUCE:**
Tester would have to have M2 Ultra Studio configured with Blackmagic Decklink video card installed in external PCIe card slot, which is most likely an uncommon setup. Tester would need to replicate same steps as mentioned above.

---

**EXPECTED BEHAVIOUR:**
Final Cut Pro should have launched normally after updating to 10.8. Was not expecting FCP to hang.

---

**SCREENSHOTS:**
xxx

---

**SPECS:**
- 2023 M2 Mac Ultra Studio, 24‑Core CPU and 60‑Core GPU, 1TB, 64GB RAM
- OWC Thunderbay Flex 8 DAS TB3 w/Blackmagic Design Decklink Mini 4K Monitor card
- macOS Sonoma 14.4.1
- Final Cut Pro 10.8
- Blackmagic Design Desktop Video 14.0.1

---

**ADDITIONAL COMMENTS:**
It seems there's a conflict with the Blackmagic Design Desktop Video app, not necessarily the external OWC Thunderbay Flex 8. However, it's odd that there was no issue using the same setup with the same apps on the M1 Mac Book Pro.

**Comment by San-Drino** [comment]
Same problem here. Same MacStudio, same Desktop Video drivers but with two Ultrastudio Thunderbolt: Mini Monitor 3g and Mini Recorder 3g. The solution for me is to force close FCP and reopen it immediately. This way it starts working properly again. As in your case FCP is not listed under camera permissions (Sometimes I use a Cam Link and have a Webcam). Of course in the MacBookPro M1, FCP regularly accesses the cameras I use . A lot of problems this version :(

**Comment by San-Drino** [comment]
Removing the user cache (~/Library/Caches/), seems to have solved the problem! It now boots normally and a/v out works as it should. At least in my system...

**Comment by FiletMyYawn** [comment]
Was it the (com.apple.FinalCut) user cache? Tried that and it didn't work unfortunately.

**Comment by San-Drino** [comment]
I deleted everything from the ~/Library/Caches/ folder, then rebooted. Make a backup of the contents before proceeding, just in case. In general, it shouldn't cause you any problems, upon reboot the cache files are automatically recreated. I don't know, these problems shouldn't be there...

**Comment by San-Drino** [comment]
I was thinking... Did you try this workaround from Apple? https://support.apple.com/en-us/108387

**Comment by FiletMyYawn** [comment]
> I deleted everything from the ~/Library/Caches/ folder, then rebooted. Make a backup of the contents before proceeding, just in case. In general, it shouldn't cause you any problems, upon reboot the cache files are automatically recreated. I don't know, these problems shouldn't be there...

Just tried that deleting all from the ~/Library/Caches/ folder, no change. Thanks for the tip though!

**Comment by FiletMyYawn** [comment]
> I was thinking... Did you try this workaround from Apple? https://support.apple.com/en-us/108387

I had done this months back and it solved the issue at the time. However, it isn't solving the issue this go around.

**Comment by San-Drino** [comment]
Yes, sorry I re-read your initial post. I'm out of ideas at this point. The only thing I would try might be a fresh installation of Sonoma with just Final Cut and desktop video (not sure if you use SOftRaid). I would make sure the OWC was connected to a Thunderbolt port without daisychaining anything. I assume the reduced security from the utilities security options is already in place. Out of curiosity, do you have any other software to try besides Final Cut? For example, DaVinci? One thing I've noticed in my system, is that the other software I use (DaVinci, Nobe Omniscope, OBS...), continue to run normally without a hitch. It seems to me that we are back to 30 years ago with the problems of Windows 95... :)

**Comment by latenitefilms** [comment]
I'll try reaching out to the Final Cut Pro team and see if it's on their radar.

**Comment by latenitefilms** [comment]
Apple says:

> After reviewing the issue you reported, we've determined it is a third-party software problem.
> If you update to Blackmagic Design's Desktop Video 14.0.1 or higher (with 14.4.1 being the latest version), the problem should no longer occur.
> 
> You can find the software updates on Blackmagic Design's website here:
> 
> https://www.blackmagicdesign.com/support/family/capture-and-playback
> 
> Please contact Blackmagic Design should you require further assistance.
